Framework/API/src/LogManager.cpp

```diff
@@ -212,17 +212,13 @@ const DateAndTime LogManager::getFirstPulseTime() const {
   // a DAS bug at SNS around Mar 2011 where the first pulse time is Jan 1, 1990."
   // There was no explanation why 100 was picked as the maximum number of times to skip.
```

Comment:
If we're no longer limiting to 100 we should probably update the comment to match.

Reply:
I've changed it so the 100 limit is still there, just in case